src/client.cpp

```diff
@@ -690,7 +686,7 @@ bool Client::loadMedia(const std::string &data, const std::string &filename)
 
 		// Create an irrlicht memory file
 		io::IReadFile *rfile = irrfs->createMemoryReadFile(
-				*data_rw, data_rw.getSize(), "_tempreadfile");
+				(void*)data.c_str(), data.size(), "_tempreadfile");
```

Comment:
Best would be:

1. Adding an #ifdef for irrlicht 1.9 with the "direct" way that doesn't do a cast

1. in the else part, add a comment that properly explains that casting is legal in this specific case

Other people might even say that the < 1.9 case should remain a copy, idk.